openupgrade_scripts/scripts/sale/17.0.1.2/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,106 @@
+---Models in module 'sale'---
+new model sale.mass.cancel.orders [transient]
+new model sale.order.discount [transient]
+# NOTHING TO DO
+
+---Fields in module 'sale'---
+sale         / account.analytic.applicability / business_domain (False)       : selection_keys is now '['bill', 'expense', 'general', 'invoice', 'manufacturing_order', 'purchase_order', 'sale_order']' ('['bill', 'expense', 'general', 'invoice', 'purchase_order', 'sale_order']')
+sale         / crm.team                 / use_quotations (boolean)      : DEL
+sale         / product.document         / attached_on (selection)       : NEW selection_keys: ['quotation', 'sale_order']
+# NOTHING TO DO
```

Comment:
```
if use_quotations  = TRUE:
   attached_on = 'quotation' 
elif exists attachments:
   attached_on = 'sale_order'
else:
   attached_on = False
```
pseudocode, but not sure how feasible is this.

Reply:
I don't think we should touch this, as it makes attachments visible in the portal, and it's a manual decision which attachments to show there